os-autoinst-obs-auto-submit

```diff
@@ -7,7 +7,7 @@ src_project=${src_project:-devel:openQA}
 dst_project=${dst_project:-${src_project}:tested}
 staging_project=${staging_project:-${src_project}:testing}
 submit_target="${submit_target:-"openSUSE:Factory"}"
-submit_target_escaped=$(echo -n "$submit_target" | sed -e 's|\:|_|g') # avoid, e.g. "repoid 'openSUSE:Factory' is illegal"
+submit_target_extra="${submit_target_extra:-"$submit_target"}"
```

Comment:
The name implies that specified here would be "extra" to "submit_target" but you rely on the default value being passed as submit_target. How about using a default value of empty string and in https://github.com/os-autoinst/scripts/pull/384/files#diff-e9e4a2f7fd0feb7816f56d2fcd9ad02870fc9344ea52dfc275699c1a7d2cbff8R21 you read from both submit_target and submit_target_extra

Reply:
sounds doable and efficient too. should I go with another name as well? maybe `submit_target_list`?

Reply:
Done. and variable renamed to `submit_target_supplemental`